src/main/java/core/basesyntax/Print.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Print {
```

Comment:
Better to rename to `Drawable` or `Paintable` or `Painter`

https://howtodoinjava.com/java/basics/java-naming-conventions/